docs/source/tools/mythxvsc/index.rst

```diff
@@ -0,0 +1,61 @@
+.. meta::
+   :description: MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
+
+.. _tools.mythxvsc:
+
+MythXvsc
+============================
+
+MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
+
+The extensions provides:
+
+- Smart contract compilation (via solidity VSCode extension)
+- Log in to MythX platform
+- AST extraction from compiled source
+- Submission of analysis
+- Displaying analysis result in VSCode in a linting fashion
+
+
+
+
+Dependencies
+-----
```

Comment:
Making the underline shorter than the text above triggers a build error. (You can see this when you build with `make html`.)